test/distributed/fsdp/test_wrap.py

```diff
@@ -395,6 +395,21 @@ def forward(self, input):
             loss.backward()
             optim.step()
 
+    @skip_if_lt_x_gpu(1)
+    def test_zero_argument(self):
+        class ZeroArguModel(nn.Module):
+            def __init__(self) -> None:
+                super().__init__()
+                self.a = torch.nn.Parameter(torch.tensor([1.0]))
+
+            def forward(self):
+                return self.a
+
+        device = torch.device("cuda")
+        model = ZeroArguModel().to(device)
+        model = FSDP(model)
+        model()
```

Comment:
nit:
```suggestion
        device = torch.device("cuda")
        model = FSDP(model, device_id=device)
        model()
```
can probably test for correctness of forward, not just that it does not error?

Reply:
Compare the forward result with expected value(torch.tensor([1.0])) here